### ISSUE TITLE
Updating to 0.11.0 breaks letsencrypt

### ISSUE THREAD

**Issue by ben5516** [Ubiquiti-App/UNMS#110]
0.11.0 update broke letsencrypt. The UNMS log file shows this:

```sh
{"source":"stdout","log":"info Updating SSL certificate on Nginx for hostname [redacted]","container_id":"dcce4769469eab320de6348c05326e7d2b0ea5e4505ef3636d22dcbaded08d47","container_name":"/unms"}
2017-12-28T18:53:31Z	unms	{"log":"error { message: 'SSL cert update failed',","container_id":"dcce4769469eab320de6348c05326e7d2b0ea5e4505ef3636d22dcbaded08d47","container_name":"/unms","source":"stdout"}
```

Any ideas on how to fix this? Or is a fix forthcoming?

**Comment by radekskrivan** [comment]
@ben5516  Hello Ben. 0.11.0. introduced internal nginx proxy which may be the cause of your situation. Here are some issues other community members reported:
https://community.ubnt.com/t5/UNMS-Beta/Let-s-Encrypt-was-turned-off-because-the-previous-attempt-had/m-p/2166324/highlight/true#M2781
https://community.ubnt.com/t5/UNMS-Beta/UNMS-Beta-0-11-0rc1-Does-not-work-with-certificates-SSL/m-p/2161498/highlight/true#M2677
Please have a look at those and if nothing help you let us know in more details what is your situation.

**Comment by radekskrivan** [comment]
@ben5516  I have a new info Ben. It seems like a bug we already encountered elsewhere. We will try to fix it asap.

**Comment by jflidr** [comment]
@ben5516 Could you please check in UNMS UI in  _SETTINGS -> UNMS -> Use Let's Encrypt_ if there is any error? There should be more information. Thank you.

**Comment by jflidr** [comment]
@ben5516 We have found a bug in Let's Encrypt certificate renewal. It will be automatically fixed during installation of UNMS 0.11.1 or you can fix your UNMS 0.11.0 with [this script](https://github.com/Ubiquiti-App/UNMS/wiki/UNMS-scripts#workaround-for-expired-lets-encrypt-certificates-in-unms-0110).

**Comment by woopstar** [comment]
This is resolved in 0.11.1
